.gitignore

```diff
@@ -0,0 +1,13 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
```

Comment:
I'm not sure if it's best practice to gitignore `Gemfile.lock`. Is that specific to a Rubygem?

Reply:
Yes, you should never include the lockfile when writing a Gem. The .gemspec takes care of dependencies and their versions.